src/llmcompressor/transformers/sparsification/compressed_tensors_utils.py

```diff
@@ -209,8 +209,9 @@ def skip(*args, **kwargs):
         save_pretrained_wrapper._overriden = True
         return save_pretrained_wrapper
 
-    # wrap save_pretrained
-    model.save_pretrained = save_pretrained_compressed(model.save_pretrained)
+    # wrap save_pretrained if not already
+    if not getattr(model.save_pretrained, "_overriden", False):
+        model.save_pretrained = save_pretrained_compressed(model.save_pretrained)
```

Comment:
Why do we need to add the additional check before updating the save_pretrained?

Reply:
There was an issue that already wrapped model was being wrapped again - I think it was for stage runners.

Reply:
Yes, there may be many consequences unintended because preprocessing happens twice. We should make sure that pathway is thought through before this merges.

Reply:
Another consequence I thought of is if the user passes a model stub, the model is essentially thrown away in between stages